py/client-ticking/README.md

```diff
@@ -33,22 +33,24 @@ you will continue to use that venv here.
 ### Install Cython in the venv
 
 If you've exited your venv, re-activate it with something like:
-```
+``` shell
 source ~/py/cython/bin/activate
 ```
 
 Then run 
-```
+``` shell
 pip3 install cython
 ```
 
 ### Build the shared library:
 
-```
-cd ${DHROOT}/py/client-ticking
+``` shell
+cd ${DHROOT}
+export DEEPHAVEN_VERSION=$(./gradlew printVersion -q)
```

Comment:
This should work. I see you don't export CFLAGS and instead do it during the build_ext command. The same can be done w/ DEEPHAVEN_VERSION for build_ext and bdist_wheel if we prefer.